flink-runtime-web/web-dashboard/src/app/pages/job/overview/flamegraph/job-overview-drawer-flamegraph.component.ts

```diff
@@ -55,7 +55,7 @@ export class JobOverviewDrawerFlameGraphComponent implements OnInit, OnDestroy {
   public selectedVertex: NodesItemCorrect | null;
   public flameGraph = {} as JobFlameGraph;
 
-  public graphType = 'on_cpu';
+  public graphType = 'on_cpu' as string;
```

Comment:
This cast is redundant. Also it should be readonly since it is only used on line 69.

A better way of doing this is to use a enum to manage the graphType rather than string:
```typescript
enum FlameGraphType {
    ON_CPU = 'on_cpu',
    OFF_CPU = 'off_cpu',
    FULL = 'full'
}
```

Reply:
Hi @yangjunhan , thanks a lot for your review.

> This cast is redundant. Also it should be readonly since it is only used on line 69.

Actually, the `graphType` isn't readonly. `nz-radio-button` will change the graph Type.

> A better way of doing this is to use a enum to manage the graphType rather than string.

Great suggestion, updated.